### ISSUE TITLE
Incorrect output from `convert_bp_list_to_dotbracket`

### ISSUE THREAD

**Issue by rhiju** [DasLab/arnie#35]
Caught by David Cox and me during Ribonanza paper writing and testing on CASP15 targets:

Here's code snippet:

```
#!/usr/bin/python3
from arnie.utils import *
from ast import literal_eval

bp = literal_eval('[[0, 362], [1, 361], [2, 360], [3, 359], [4, 358], [5, 357], [6, 356], [7, 355], [8, 354], [9, 26], [10, 25], [11, 24], [12, 23], [13, 22], [14, 21], [15, 20], [27, 71], [28, 70], [29, 69], [30, 68], [32, 66], [33, 65], [34, 64], [35, 63], [36, 62], [37, 61], [38, 53], [39, 52], [40, 51], [41, 50], [42, 49], [54, 135], [55, 134], [56, 133], [57, 132], [58, 131], [59, 130], [72, 353], [73, 352], [74, 298], [75, 297], [76, 296], [77, 295], [78, 294], [79, 293], [80, 292], [81, 291], [82, 290], [83, 289], [84, 288], [86, 286], [87, 285], [89, 283], [95, 280], [102, 265], [103, 264], [104, 263], [106, 261], [107, 260], [108, 259], [109, 258], [110, 257], [111, 256], [112, 255], [113, 254], [114, 253], [115, 149], [116, 148], [118, 146], [119, 145], [120, 144], [121, 143], [123, 141], [124, 140], [125, 139], [126, 138], [127, 137], [150, 167], [151, 166], [152, 165], [153, 164], [154, 163], [155, 162], [156, 161], [168, 252], [169, 251], [170, 187], [171, 186], [172, 185], [173, 184], [174, 183], [175, 182], [176, 181], [188, 205], [189, 204], [190, 203], [191, 202], [192, 201], [193, 200], [194, 199], [206, 250], [207, 249], [208, 248], [210, 246], [212, 244], [213, 243], [214, 242], [215, 241], [216, 240], [217, 232], [218, 231], [219, 230], [220, 229], [221, 228], [233, 319], [234, 318], [235, 317], [236, 316], [237, 315], [238, 314], [299, 333], [300, 332], [301, 331], [302, 330], [303, 329], [304, 328], [305, 327], [307, 325], [308, 324], [309, 323], [310, 322], [311, 321], [334, 351], [335, 350], [336, 349], [337, 348], [338, 347], [339, 346], [340, 345]]')
dbn = convert_bp_list_to_dotbracket(bp,363)
print(dbn)
```

The output is:

```
((((((((((((((((....)))))))((((.(((((((((((......)))))[[[[[[.)))))).))))(((((((((((((.((.(.....(......(((.(((((((((((.((((.(((((..]]]]]].))))).)))).))(((((((....)))))))(((((((((....)))))))(((((((....)))))))(((.(.((((((((((......))))).......))))).).)))))))))))))).)))..............)..).)).)))))))))))(((((((.(((((.........))))).)))))))(((((((....))))))))))))))))))
```

But it should be:

```
((((((((((((((((....)))))))((((.(((((((((((......)))))[[[[[[.)))))).))))(((((((((((((.((.(.....(......(((.(((((((((((.((((.(((((..]]]]]].))))).)))).))(((((((....)))))))(((((((((....)))))))(((((((....)))))))(((.(.((((((((((......)))))[[[[[[.))))).).)))))))))))))).)))..............)..).)).)))))))))))(((((((.(((((..]]]]]].))))).)))))))(((((((....))))))))))))))))))
```

That is, the second kissing loop (`[[[[[[`,`]]]]]]`) is missing.

Tagging @rkretsch for advice or a hotfix, and @tkaragianes for help with then updating `pip` (which doesn't seem to have pk_predictors at all?)

**Comment by rkretsch** [comment]
fixed https://github.com/DasLab/arnie/pull/36

**Comment by rkretsch** [comment]
merged, leaving open for @tkaragianes in case anything for pip needs to be done

**Comment by rhiju** [comment]
@tkaragianes this was resolved through #36  but we left open to make sure we got into Pypi. Did we get the PK branch into the pip-installable ARNIE? If so, please close this issue!

**Comment by tkaragianes** [comment]
Closed with #37.
